### PR TITLE
feat: generate CLI README.md

### PR DESCRIPTION
**Pull request by blacha**
#### Motivation

I do not like having to update the same information in two places

#### Modification

Add script to generate a README.md for all CLIs that are registered in the `AllCommands`
Generate the READMEs on every push to master to validate they have not been modified.

#### Checklist

_If not applicable, provide explanation of why._

- [ ] Tests updated
- [ ] Docs updated
- [ ] Issue linked in Title

### CODE REVIEW

File: src/commands/copy/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## Usage
+
+copy <options> [...location]
+
+### Arguments
+
+| Usage         | Description              | Options |
+| ------------- | ------------------------ | ------- |
+| [...location] | Manifest of file to copy |         |
```
Comment (l0b0) [inline src/commands/copy/README.md:13]: `[...foo]` is not a syntax I'm familiar with. What's meant here? Is it zero-or-one locations? If so, `[location]` is more common syntax for that. If it's exactly one location, that would normally be `<location>`.

File: src/commands/stac-validate/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+| Usage         | Description                            | Options |
+| ------------- | -------------------------------------- | ------- |
+| [...location] | Location of the STAC files to validate |         |
+
+### Options
+
+| Usage                  | Description                            | Options  |
+| ---------------------- | -------------------------------------- | -------- |
+| --config <str>         | Location of role configuration file    | optional |
```
Comment (l0b0) [inline src/commands/stac-validate/README.md:19]: Aren't all options and flags optional?

**Review by l0b0 — COMMENTED** [review]
Love it!

File: src/commands/copy/README.md (hunk 1 above)
Comment (blacha) [reply src/commands/copy/README.md:13]: this is just the output of the help text, I would imagine it means `string[]` as `...` is the javascript spread operator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

File: src/commands/stac-validate/README.md (hunk 2 above)
Comment (blacha) [reply src/commands/stac-validate/README.md:19]: there are quite a few that are not optional?

File: src/commands/copy/README.md (hunk 1 above)
Comment (l0b0) [reply src/commands/copy/README.md:13]: So we want to use that syntax? OK.

**Review by l0b0 — APPROVED** [review]

**Pull request merged**